tests/system/action/motion/test_create_forwarded.py

```diff
@@ -63,6 +64,7 @@ def test_correct_origin_id_set(self) -> None:
                 "all_origin_ids": [12],
             },
         )
+        assert model.get("forwarded")
```

Comment:
Would you please use the optional field reason to check it overwrites the motions reasons

Reply:
Added a test for 'reason'.